blade-graphics/src/metal/resource.rs

```diff
@@ -181,7 +181,11 @@ impl crate::traits::ResourceDevice for super::Context {
                 if desc.array_layer_count > 1 {
                     metal::MTLTextureType::D2Array
```

Comment:
same here - do we want to leave this for a followup? if so, let's leave a warning

Reply:
Didn't check but yup easily supported case, will add